src/i18n-cache/index.js

```diff
@@ -0,0 +1,172 @@
+/* eslint-disable no-console */
+
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const path = require( 'path' );
+const fetch = require( 'node-fetch' );
+
+const supportedLocales = [
+	'ar', // Arabic
+	'bg', // Bulgarian
+	'bo', // Tibetan
+	'ca', // Catalan
+	'cs', // Czech
+	'cy', // Welsh
+	'da', // Danish
+	'de', // German
+	'en-au', // English (Australia)
+	'en-ca', // English (Canada)
+	'en-gb', // English (UK)
+	'en-nz', // English (New Zealand)
+	'en-za', // English (South Africa)
+	'el', // Greek
+	'es', // Spanish
+	'es-ar', // Spanish (Argentina)
+	'es-cl', // Spanish (Chile)
+	'es-cr', // Spanish (Costa Rica)
+	'fa', // Persian
+	'fr', // French
+	'gl', // Galician
+	'he', // Hebrew
+	'hr', // Croatian
+	'hu', // Hungarian
+	'id', // Indonesian
+	'is', // Icelandic
+	'it', // Italian
+	'ja', // Japanese
+	'ka', // Georgian
+	'ko', // Korean
+	'nb', // Norwegian (Bokmål)
+	'nl', // Dutch
+	'nl-be', // Dutch (Belgium)
+	'pl', // Polish
+	'pt', // Portuguese
+	'pt-br', // Portuguese (Brazil)
+	'ro', // Romainian
+	'ru', // Russian
+	'sk', // Slovak
+	'sq', // Albanian
+	'sr', // Serbian
+	'sv', // Swedish
+	'th', // Thai
+	'tr', // Turkish
+	'uk', // Ukrainian
+	'ur', // Urdu
+	'vi', // Vietnamese
+	'zh-cn', // Chinese (China)
+	'zh-tw', // Chinese (Taiwan)
+];
+
+const getLanguageUrl = ( locale, plugin ) =>
```

Comment:
This file is almost the same as the one we have in [`gutenberg/packages/react-native-editor/i18n-cache`](https://github.com/WordPress/gutenberg/blob/trunk/packages/react-native-editor/i18n-cache/index.js) but here I included the `plugin` parameter that will be useful for when we introduce the translations for Jetpack.